tests/end_to_end/conftest.py

```diff
@@ -33,9 +33,24 @@ def mock_django_feature(mock_django_project):
     )
 
 
+@pytest.fixture()
+def mock_nested_environment(mock_django_project, mock_django_feature):
```

Comment:
I'm not sure what we mean by 'nested environment' ?

Reply:
It's because the mock class that we have: https://github.com/Flagsmith/flagsmith-engine/pull/150/files#diff-678ec674d356b7a4504b6afb9760ee1819592bc4d3aa8667282b9ef321131b6dL151 does not have environment as an attribute, and using the environment fixture will create a recursive loop

Reply:
Can we add a comment about this then? Similar to the one that we have on the `mock_django_segment` fixture below I guess.